docs/wiki/Deploying-Enterprise-Scale-CustomerUsage.md

```diff
@@ -0,0 +1,25 @@
+<!-- markdownlint-disable -->
+## Telemetry Tracking Using Customer Usage Attribution (PID)
```

Comment:
Need to add this file into the following locations as links:

1. Wiki Sidebar 
2. Wiki Home
3. Repo root README.md

Reply:
Added